### PR TITLE
topology: add demux pipeline to CML max98357a tplg

### PR DESCRIPTION
**Pull request by keqiaozhang**
This topology will add a mux pipeline to support echo reference for CML.

Signed-off-by: Zhang Keqiao <keqiao.zhang@linux.intel.com>

### CODE REVIEW

**Comment by paulstelian97** [comment]
@tlauda I think it's included from sof-cml-demux-rt5682.m4

File: tools/topology/sof-cml-demux-rt5682-max98357a.m4 [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+# Topology for Cometlake with rt5682 headset on SSP0, max98357a spk on SSP1
+#
+# Adding max98357a spk on SSP1 on top of sof-cml-rt5682.
+#
+
+# Include SOF CML RT5682 Topology
+# This includes topology for RT5682, DMIC and 3 HDMI Pass through pipeline
+include(`sof-cml-demux-rt5682.m4')
+
+DEBUG_START
```
Comment (tlauda) [inline tools/topology/sof-cml-demux-rt5682-max98357a.m4]: What is this? Most of the topologies don't have it.

Comment (keqiaozhang) [reply tools/topology/sof-cml-demux-rt5682-max98357a.m4]: It defined in m4/debug.m4 and used for m4 macros debug.
@juimonen , could you give us more detailed explanations here?

Comment (lgirdwood) [reply tools/topology/sof-cml-demux-rt5682-max98357a.m4]: @keqiaozhang it's only needed for debugging so it can be removed.

Comment (keqiaozhang) [reply tools/topology/sof-cml-demux-rt5682-max98357a.m4]: thanks @lgirdwood, this debugging is removed.

**Pull request merged**